homeassistant/components/barry/sensor.py

```diff
@@ -0,0 +1,63 @@
+"""Platform for sensor integration."""
+from datetime import timedelta
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(hours=1)
+MIN_TIME_BETWEEN_UPDATES = timedelta(hours=1)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Barry sensor."""
+    barry_connection = hass.data.get(DOMAIN)
+    price_code = hass.data.get("priceCode")
+    async_add_entities([BarrySensor(barry_connection, price_code)], True)
+
+
+class BarrySensor(Entity):
+    """Representation of a Sensor."""
+
+    def __init__(self, barry_home, price_code):
+        """Initialize the sensor."""
+        self._barry_home = barry_home
+        self.price_code = price_code
+        self._state = None
+        self._name = "Barry"
+        self._device_state_attributes = {}
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        return self._device_state_attributes
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return f"Electricity price {self._name}"
+
+    @property
+    def state(self):
+        """Return the state of the sensor."""
+        return self._state
+
+    @property
+    def model(self):
+        """Return the model of the sensor."""
+        return "Price Sensor"
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit of measurement of this entity."""
+        return "DKK/kWh"
```

Comment:
Two units of measurement? Can the lib/API return the correct value for this?

Reply:
@timmo001 "DKK/kWh" is the unit of measurement which is static for now and therefore I hardcoded it.